README.md

```diff
@@ -162,6 +162,22 @@ $ ruby start_finish_trace.rb
 
 **Busted** includes an [example `dtrace` probe](/dtrace/probes/examples/method-cache-clear.d) for use on the command line or an application.  See the [probe](/dtrace/probes/examples/method-cache-clear.d) for usage.
 
+# Rails Usage
```

Comment:
Please, let's make this a subheadline via `## Rails Usage`, so it [matches the rest of the file](https://github.com/HParker/busted/blob/652cfd30e482f0c8fe2abc3fb635a716c6d0aa7a/README.md#advanced-usage).